docker/develop/develop.dockerfile

```diff
@@ -48,15 +67,14 @@ RUN zypper install --no-recommends -y \
     vim                        \
     wget                       \
     which                      \
-    xorriso
+    xorriso                    \
+    && zypper clean
```

Comment:
Why is that needed?

Reply:
See https://github.com/hadolint/hadolint/wiki/DL3036

Reply:
Thanks! I didn't know hadolint before, IMO it wouldn't hurt to have an explanation inside the dockerfile.

Reply:
Will add a comment about it.